icssc/AntAlmanac
# Fix install script to work on Windows

## Summary
The previous `server-install` command did not work on Windows.  

Credit to @falsidge to finding and fixing this: 
> this uses test which only works in linux
> so the project currently won't install the dependencies on windows 
> ```
> "client-install": "npm install ./client",
> "server-install": "npm install ./antalmanac-backend", 
> ```
> should work

## Test Plan
- Run `npm i` on a workspace **without** the backend recursively installed. Verify that the client still installs correctly and runs without the backened
- Run `npm i` on a workspace **with** the backend recursively installed. Verify that both install and run correctly.

## Issues
Update for #218 

## Future Followup 
If this user didn't recursively install the backend, this will (correctly) fail. However the error message doesn't help the user understand why it failed. It should tell them that they need to recursively install the backend.
![image](https://user-images.githubusercontent.com/29494270/156083003-f27fd104-9e9f-4581-8368-22f5091876e8.png)
